docs/guides/technical-docs-seo-guide.rst

```diff
@@ -0,0 +1,278 @@
+as well as improvements provided by Read the Docs itself.
+
+
+SEO Basics
+----------
+
+Search engines like Google and Bing crawl through the internet
+following links in an attempt to understand and build an index
+of what various pages and sites are about.
+This is called "crawling" or "indexing".
+When a person sends a query to a search engine,
+the search engine evaluates this index using a number of factors
+and attempts to return the results most likely to answer that person's question.
+
+How search engines "rank" sites based on a person's query
+is part of their secret sauce.
+While some search engines publish the basics of their algorithms
+(see Google's published details on PageRank), few search engines
+give all of the details in an attempt to prevent users from gaming the rankings
+with low value content which happens to rank well.
+
+Both `Google <https://support.google.com/webmasters/answer/35769>`_
+and `Bing <https://www.bing.com/webmaster/help/webmaster-guidelines-30fba23a>`_ publish a set of guidelines
+to help make sites easier to understand for search engines and rank better.
+To summarize some of the most important aspects
+as they apply to technical documentation, your site should:
+
+* Use descriptive and accurate titles in the HTML ``<title>`` tag.
+* Ensure your URLs are descriptive. They are displayed in search results.
+* Avoid low content pages or pages with very little original content.
+* Avoid tactics that attempt to increase your search engine ranking
+  without actually improving content.
+* Google specifically `warns about automatically generated content`_
+  although this applies primarily to keyword stuffing and low value content.
+  High quality documentation generated from source code
+  (eg. auto generated API documentation) seems OK.
+
+  .. _warns about automatically generated content: https://support.google.com/webmasters/answer/2721306
+
+While both Google and Bing discuss site performance as an important factor
+in search result ranking, this guide is not going to discuss it in detail.
+Most technical documentation that uses Sphinx or Read the Docs
+generates static HTML and the performance is typically decent
+relative to most of the internet.
+
+
+Optimizing your docs for search engine spiders
+----------------------------------------------
+
+Once a crawler or spider finds your site, it will follow links and redirects
+in an attempt to find any and all pages on your site.
+While there are a few ways to guide the search engine in its crawl
+for example by using a sitemap or a robots.txt file which we'll discuss shortly,
+the most important thing is making sure the spider can follow links on your site
+and get to all your pages.
+
+Avoid orphan pages
+~~~~~~~~~~~~~~~~~~
+
+Sphinx calls pages that don't have links to them "orphans"
+and will throw a warning while building documentation that contains an orphan
+unless the warning is silenced with the :ref:`orphan directive <sphinx:metadata>`:
```

Comment:
For some reason, this `:ref:` is rendering as https://sphinx.readthedocs.io/en/latest/usage/restructuredtext/field-lists.html#metadata instead of https://sphinx-doc.org

Reply:
This is due to this: https://github.com/readthedocs/readthedocs.org/blob/9f928343b967247d9bed8f5d7d851e38cdeef131/docs/conf.py#L59

I can test out changing it.